Archivos.c

```diff
@@ -0,0 +1,66 @@
+#include"Archivos.h"
+
+/*1)*/Archivos::Archivos(){
+    //ctor sin patamentros
+        abierto=false;
+
+}
+
+/*2)*/Archivos::Archivos(string camino){
+    //ctor
+    abrirArchivo(string camino);
+    if(this->archivo.good()){
+        abirto=true;
+    }else{
+        abierto=false;
+    }
+
+}
+
+/*3)*/Archivos::~Archivos(){
+    //dtor
+    if(this->estaAbierto()){
```

Comment:
Dentro de la clase no hace falta usar el metodo, directamente se puede usar el atributo
```suggestion
    if(this->abierto) {
```